pkg/jiraloader/jiraloader.go

```diff
@@ -29,6 +30,16 @@ const jiraTimeLayout = "2006-01-02T15:04:05.000Z0700"
 
 func (jl *JIRALoader) LoadJIRAIncidents() error {
 	start := time.Now()
+	log.Infof("populating jira incident cache...")
+	var dbIssues []string
+	jl.dbc.DB.Table("jira_incidents").Pluck("key", &dbIssues)
```

Comment:
If I understand it correctly this will get all of the keys from the table each time, I guess when I was thinking about it I was thinking we would only want entries without resolution_time.  Just wondering about the 5 years from now case or if we move to another system and don't have historical context, etc.

Reply:
5 years from now even if you had 5 incidents a day, 7 days a week you'd have under 10K rows.  Even at 100,000 incidents this cache would be at most 4MB, it's not really worth optimizing.

Reply:
For resolution_time, what if you unresolve an incident which I think we've done in the past?